src/Migrations/Version20180412141456.php

```diff
@@ -0,0 +1,109 @@
+
+        $tagTable = $schema->createTable('symfony_demo_tag');
+        self::createIdentifier($tagTable);
+        self::createVarchar($tagTable, 'name');
+        $tagTable->addUniqueIndex(['name'], 'UNIQ_4D5855405E237E06');
+
+        $postTable = $schema->createTable('symfony_demo_post');
+        self::createIdentifier($postTable);
+        self::createVarchar($postTable, 'title');
+        self::createVarchar($postTable, 'slug');
+        self::createVarchar($postTable, 'summary');
+        $postTable->addColumn('content', Type::TEXT);
+        $postTable->addColumn('published_at', Type::DATETIME);
+        $postTable->addColumn('author_id', Type::INTEGER)->setNotnull(true);
+        $postTable->addForeignKeyConstraint($userTable, ['author_id'], ['id'], [], 'FK_58A92E65F675F31B');
+
+        $commentTable = $schema->createTable('symfony_demo_comment');
+        self::createIdentifier($commentTable);
+        $commentTable->addColumn('post_id', Type::INTEGER)->setNotnull(true);
+        $commentTable->addColumn('author_id', Type::INTEGER)->setNotnull(true);
+        $commentTable->addColumn('content', Type::TEXT)->setNotnull(true);
+        $commentTable->addColumn('published_at', Type::DATETIME)->setNotnull(true);
+        $commentTable->addForeignKeyConstraint($postTable, ['post_id'], ['id'], [], 'FK_53AD8F834B89032C');
+        $commentTable->addForeignKeyConstraint($userTable, ['author_id'], ['id'], [], 'FK_53AD8F83F675F31B');
+
+        $postTagTable = $schema->createTable('symfony_demo_post_tag');
+        $postTagTable->addColumn('post_id', Type::INTEGER)->setNotnull(true);
+        $postTagTable->addColumn('tag_id', Type::INTEGER)->setNotnull(true);
+        $postTagTable->setPrimaryKey(['post_id', 'tag_id']);
+        $postTagTable->addForeignKeyConstraint($postTable, ['post_id'], ['id'], ['onDelete' => 'CASCADE'], 'FK_6ABC1CC44B89032C');
+        $postTagTable->addForeignKeyConstraint($tagTable, ['tag_id'], ['id'], ['onDelete' => 'CASCADE'], 'FK_6ABC1CC4BAD26311');
+    }
+
+    public function down(Schema $schema): void
+    {
+        $postTable = $schema->getTable('symfony_demo_post');
+        $postTable->dropIndex('FK_58A92E65F675F31B');
+
+        $commentTable = $schema->getTable('symfony_demo_comment');
+        $commentTable->dropIndex('FK_53AD8F834B89032C');
+        $commentTable->dropIndex('FK_53AD8F83F675F31B');
+
+        $postTagTable = $schema->getTable('symfony_demo_post_tag');
+        $postTagTable->dropIndex('FK_6ABC1CC44B89032C');
+        $postTagTable->dropIndex('FK_6ABC1CC4BAD26311');
+
+        $schema->dropTable('symfony_demo_user');
+        $schema->dropTable('symfony_demo_tag');
+        $schema->dropTable('symfony_demo_post');
+        $schema->dropTable('symfony_demo_comment');
+        $schema->dropTable('symfony_demo_post_tag');
+    }
+
+    public function postUp(Schema $schema): void
+    {
+        $this->container->get(MigrationInitialImport::class)->importInitialData();
```

Comment:
It's a bad practice to use entities in the migrations (entity can be deleted in the future). Here you use entities implicitly. You should load fixtures by raw SQL.